bin/post-create-project.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+if (! file_exists('.env')) {
```

Comment:
Remove this file, because:
- `.env.dist` file no longer exists, now we use `.env` file for default (the new symfony best practices)
- `public/.gitignore` file is bad practices, see previous comment
- `README_project.md` file no longer exists, because we should only have one `README.md` file in the project so as not to lose users

Reply:
> .env.dist file no longer exists, now we use .env file for default (the new symfony best practices) 

Ok, that's correct.. Will update it. 

> public/.gitignore file is bad practices, see previous comment

That needs to stay. Otherwise it'll break the way it's currently used. In my opinion this should be here (so the files in `public` don't get added to git, but then it's removed after running `composer create-project` on it. 

> README_project.md file no longer exists, because we should only have one README.md file in the project so as not to lose users

But, it does need to exist.. This project has a specific README, but when the project is installed using `composer create-project`, it should not have _this_ README, but one that's suitable for that project.